docs/contributing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contributing
+type: docs
+menu: docs
+---
+# Contributing
```

Comment:
should we mention somewhere that we have slack and want people to interact with us there? Also wdyt about having all this in this form https://help.github.com/en/articles/setting-guidelines-for-repository-contributors or either linking one to the other?

Reply:
There's actually already a link to our community Slack a little further down, and I've also now linked to our existing contributing guidelines as well.